### PR TITLE
Document wrangler r2 bucket lifecycle command (list, add, remove, set)

### PR DESCRIPTION
**Pull request by jonesphillip**
Document wrangler  r2 bucket lifecycle command: list, add, remove, set

Relates to: https://github.com/cloudflare/workers-sdk/pull/7207

### CODE REVIEW

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 1]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
 5. Fill out the fields for the new rule.
 6. When you are done, select **Add rule**.
 
+### Wrangler
+
+#### Set up Wrangler
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx]:
```suggestion

```

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 2]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
...
+
+To begin, install [`npm`](https://docs.npmjs.com/getting-started). Then [install Wrangler, the Developer Platform CLI](/workers/wrangler/install-and-update/).
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx]:
```suggestion
1. Install [`npm`](https://docs.npmjs.com/getting-started). 
2. Install [Wrangler, the Developer Platform CLI](/workers/wrangler/install-and-update/).
```

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 3]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
...
+
+#### Add a lifecycle rule
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx]:
```suggestion

```

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 4]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
...
+
+Log in to Wrangler with the [`wrangler login` command](/workers/wrangler/commands/#login). Then add a lifecycle rule to your bucket by running the [`r2 bucket lifecycle add` command](/workers/wrangler/commands/#lifecycle-add).
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx]:
```suggestion
3. Log in to Wrangler with the [`wrangler login` command](/workers/wrangler/commands/#login). 
4. Add a lifecycle rule to your bucket by running the [`r2 bucket lifecycle add` command](/workers/wrangler/commands/#lifecycle-add).
```

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 5]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
...
+
+```sh
+npx wrangler r2 bucket lifecycle add <BUCKET_NAME> [OPTIONS]
+```
+
+#### Set the lifecycle configuration
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx]:
```suggestion

```

File: src/content/docs/r2/buckets/object-lifecycles.mdx [hunk 6]
```diff
@@ -31,9 +33,33 @@ When you create an object lifecycle rule, you can specify which prefix you would
...
+
+Alternatively you can set the entire lifecycle configuration for a bucket from a JSON file using the [`r2 bucket lifecycle set` command](/workers/wrangler/commands/#lifecycle-set).
```
Comment (dcpena) [inline src/content/docs/r2/buckets/object-lifecycles.mdx:47]:
```suggestion
5. Alternatively you can set the entire lifecycle configuration for a bucket from a JSON file using the [`r2 bucket lifecycle set` command](/workers/wrangler/commands/#lifecycle-set).
```

**Review by irvinebroque — APPROVED** [review]

**Review by dcpena — APPROVED** [review]

**Comment by HAKSENGVideo** [comment]
Good

**Pull request merged**